dev/packages/alpha/system/0.0.3/manifest.yml

```diff
@@ -0,0 +1,43 @@
+format_version: 1.0.0
+name: system
+title: System
+version: 0.0.1
```

Comment:
@fearful-symmetry You need to update this line to pass CI

Reply:
Ah!